README.md

```diff
@@ -292,6 +292,40 @@ const indexPost = async ({ text, id, title, published, author }) => {
 };
 ```
 
+If `MOCK_ELASTIC=1` is set in the environment variables, the `Elastic()` function creates a `mock` object and creates an instance of [ElasticSearch-mock](https://www.npmjs.com/package/@elastic/elasticsearch-mock) for testing.
+
+```js
+const { mock } = Elastic();
+
+process.env.MOCK_ELASTIC = 1;
```

Comment:
You can remove this, it would have to happen before this code was called.

Reply:
Oh right, I forgot the Jest configs